dynamax/linear_gaussian_ssm/inference.py

```diff
@@ -7,7 +7,8 @@
 
 from tensorflow_probability.substrates.jax.distributions import (
     MultivariateNormalDiagPlusLowRankCovariance as MVNLowRank,
-    MultivariateNormalFullCovariance as MVN)
+    MultivariateNormalFullCovariance as MVN,
+)
```

Comment:
There are a quite few lines of changes introduced by applying black to the modified files. Maybe it's better to first merge a separate PR of applying black, to make the diff easier to read here.

Reply:
I created a PR for formatting here: #337 so feel free to use it if you feel like it. (Please close it otherwise!)